test/e2e/context.ts

```diff
@@ -66,7 +66,14 @@ export class TestContext {
       this._clearAppUserData();
     }
 
-    const childProcess = spawn(this._electronPath, [this._appPath], { env });
+    const args = [this._appPath];
+
+    // Older versions of Electron no longer work correctly on 'ubuntu-latest' with sandbox
+    if (process.platform === 'linux') {
+      args.push('--no-sandbox');
```

Comment:
shouldn't we just do this only for older electron versions? (add a `major >= 13`)

Reply:
Changed so it only disables sandbox for < v13